instance/models/utils.py

```diff
@@ -52,6 +55,25 @@ def format_help_text(help_text):
     return " ".join(help_text.replace('\n', ' ').split())
 
 
+def _get_setting(name):
+    """
+    Return the Django setting with the given name.
+
+    This function is a helper for default_setting.  It is necessary because Django can't serialize
+    `Settings` objects.
+    """
+    return getattr(settings, name)
+
+
+def default_setting(name):
+    """
+    Return a callable that returns the Django setting with the given name.
+
+    This is useful for default values in model fields.
+    """
+    return functools.partial(_get_setting, name)
```

Comment:
@bradenmacdonald I've added the assertion, but can't use a lambda for the same reason as above.  We could make pylint happy by returning a named closure instead of a lambda, but Django would not be able to serialize this.